.github/workflows/regression-report.yml

```diff
@@ -73,30 +109,35 @@ jobs:
               content: 'rocket'
             })
 
-      - name: Checkout repository
-        uses: actions/checkout@v4
+      - name: Restore cached checkouts
+        uses: actions/cache@v4
+        with:
+          path: |
+            .
+            .ci-temp/checkstyle
+          key: ${{ runner.os }}-checkouts-${{ github.sha }}
 
       - name: Parse comment
         id: parse
         env:
           COMMENT_BODY: ${{ github.event.comment.body }}
         run: |
-          CONFIG_BUNDLE_PATH=$(echo "$COMMENT_BODY" | \
-            sed -E 's/.*(Github|GitHub), generate report for //g')
-          echo "CONFIG_BUNDLE_PATH=$CONFIG_BUNDLE_PATH"
-          ./.ci/append-to-github-output.sh "config_bundle_path" "$CONFIG_BUNDLE_PATH"
-
-      - name: Set config and projects links
-        id: set_links
-        run: |
-          BASE_URL="https://raw.githubusercontent.com/checkstyle/test-configs/main"
-          CONFIG_BUNDLE_PATH="${{ steps.parse.outputs.config_bundle_path }}"
-          CONFIG_LINK="$BASE_URL/$CONFIG_BUNDLE_PATH/config.xml"
-          PROJECTS_LINK="$BASE_URL/$CONFIG_BUNDLE_PATH/list-of-projects.properties"
-          echo "CONFIG_LINK=$CONFIG_LINK"
-          echo "PROJECTS_LINK=$PROJECTS_LINK"
-          ./.ci/append-to-github-output.sh "config_link" "$CONFIG_LINK"
-          ./.ci/append-to-github-output.sh "projects_link" "$PROJECTS_LINK"
+          if [[ "$COMMENT_BODY" =~ ^Git[Hh]ub,\ generate\ report\ for\ configs\ in\ PR\ \
+          description$ ]]; then
```

Comment:
try:
```
if [[ "$COMMENT_BODY" =~ '^Git[Hh]ub, generate report for configs in PR description$' ]];
then
```
apply to other to avoid `\ ` and wrapping of line in middle of regexp

Reply:
done

Reply:
>apply to other 

not done. please do

Reply:
I had tried it, with regex, escapes are required. I will try it again and show you the issue. It fails without regex. Escapes are required i guess. But, will test it and show.

Reply:
use single quotes

Reply:
https://github.com/checkstyle-gsoc/checkstyle/actions/runs/10539003997/job/29202119403

Reply:
> use single quotes

Failing with single quotes too. 
https://github.com/checkstyle-gsoc/checkstyle/actions/runs/10539003997/job/29202119403

Reply:
```
github/workflows/regression-report.yml:124:9: shellcheck reported issue in this script: SC2076:warning:1:26: Remove quotes from right-hand side of =~ to match as a regex rather than literally [shellcheck]
    |
12[4](https://github.com/checkstyle/checkstyle/actions/runs/10538944584/job/29201986926?pr=15519#step:4:5) |         run: |
    |         ^~~~
```

Ok.
`` ^Git[Hh]ub,' generate report for'``
Or
```
$ exp='".*"'
$ [[ $foo =~ $exp ]] && echo $BASH_REMATCH
"Hello World!"
```
We need to avoid escaping or space.